lib/ansible/modules/cloud/amazon/elb_application_lb.py

```diff
@@ -68,6 +68,12 @@
         not set then listeners will not be modified
     default: yes
     choices: [ 'yes', 'no' ]
+  purge_rules:
+    description:
+      - If yes, existing rules will be purged from the ELB to match exactly the I(Rules) defined in each I(listeners) parameter.
+    default: yes
```

Comment:
I'd rather this default to `no`. It is a change, but users are more likely to come a cropper on this being true than false.

Reply:
Ok for me. I tried to keep the same behavior that it had before the PR. @wimnat any thought?

Reply:
My 2 cts on this. I think it should have same behaviour as `purge_listeners`, meaning having a default value `yes`.
Though, I'll definitely use this feature and set it to no!

Reply:
I always consider ansible to be first and foremost trying to attain desired state which is why I default to yes. It's a mixture across other modules. I'm not too fussed but whatever the decision it should be documented so we can get some consistency

Reply:
And I always remember this tweet: https://twitter.com/stigglor/status/888221882360582145